src/rules/converters/one-line.ts

```diff
@@ -0,0 +1,17 @@
+import { RuleConverter } from "../converter";
+
+export const CheckAllTokensMsg = "ESLint's brace-style will check all tokens.";
+
+export const convertOneLine: RuleConverter = (tslintRule) => {
+    return {
+        rules: [
+            {
+                ...(tslintRule.ruleArguments.length !== 0 && {
```

Comment:
Is this logic backwards? From my understanding of https://palantir.github.io/tslint/rules/one-line and https://github.com/typescript-eslint/tslint-to-eslint-config/issues/410#issuecomment-619482577, TSLint will only check syntax constructs when enabled as a rule argument. 

Assuming that's true, there should be three cases here:

* If `one-line` has _no_ optional params enabled, it's effectively doing nothing, and can be ignored
* If `one-line` has _some_ optional params enabled, the `notice` should mention the other syntax constructs will now be linted
* If `one-line` has _all_ optional params enabled, there shouldn't be a notice*
    * *...unless there are syntax constructs ESLint catches that TSLint didn't? In which case there should be a notice for them.

Reply:
The logic seems to make sense as the current TSLint description made me think the rule was always active and the options were used to **only** check the specified tokens.

Therefore, I made the changes while offline, right now had no internet at all 😆